pimd/pim_igmpv2.c

```diff
@@ -127,6 +127,13 @@ int igmp_v2_recv_report(struct gm_sock *igmp, struct in_addr from,
 
 	memcpy(&group_addr, igmp_msg + 4, sizeof(struct in_addr));
 
+	if (to.s_addr != group_addr.s_addr) {
```

Comment:
I don't understand why these has to match? The group addr has nothing to do with the igmp packet destination. Can you explain?